mittab/libs/tests/tab_logic/test_outround_pairing.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+from django.test import TestCase
+import pytest
+
+from mittab.apps.tab.models import *
+from mittab.libs import outround_tab_logic
+
+
+@pytest.mark.django_db
+class TestOutroundPairingLogic(TestCase):
+    fixtures = ["testing_finished_db"]
+    pytestmark = pytest.mark.django_db
+
+    def generate_checkins(self):
+        for r in Room.objects.all():
+            RoomCheckIn.objects.create(room=r,
+                                       round_number=0)
+
+    def test_break(self):
+        self.generate_checkins()
+
+        outround_tab_logic.perform_the_break()
```

Comment:
Ideally we would be testing more than just that the code runs without errors -- we should be validating the pairings generated. Also would be nice to have:

1. A test which iterates through the outrounds, inputting results and confirming the correct results of the next round

2. One which starts with partials

Reply:
Yep, I’ll get on that — I’m always lazy with unit tests haha

Reply:
Newest commit does both these things.